model/span.go

```diff
@@ -111,6 +112,13 @@ type DestinationService struct {
 	Resource string
 }
 
+// Composite holds details on a group of spans compressed into one.
+type Composite struct {
+	Count               int
+	Sum                 float64
```

Comment:
would you mind adding a comment to Sum (at least), noting that it is in milliseconds?